Tests/TestHelpers.swift

```diff
@@ -1,12 +1,12 @@
 import Moya
-import Foundation
 
-extension String {
-    var urlEscaped: String {
-        return self.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
-    }
-}
+#if os(OSX)
+import AppKit
+#else
+import Foundation
```

Comment:
@SD10 `GCFloat` is actually imported in `CoreGraphics`, which in turn is not included in `Foundation`, but is included in `UIKit`. :)